src/library/modules/Service.js

```diff
@@ -483,6 +483,26 @@ See Manifest File [manifest.json] under "background" > "scripts"
 		}
 	});
 	
+	/* On Chrome Storage Changed
+	Sync localStorage parts with Chrome Storage
+	Used for sync quests data on different machines
+	------------------------------------------*/
+	chrome.storage.onChanged.addListener(function(changes, namespace) {
+		for (var key in changes) {

```

Comment:
That was used for checking all keys and changes. Now it can be replaced with such simple check. Thank you.